### PR TITLE
Added three new events to activityHub.js (incoming-call-created,incom…

### PR DESCRIPTION
**Pull request by 93Sprinter**
…ing-call-connecting,incoming-call-disconnecting).

These three events are handled inside the index.js of minAppWindow to run a command on an incoming call. The command can be specified as cli argument or inside a config.json as incomingCallCommand parameter. As an example this can be used to play a sound on a second speaker to simulate a second ringer.

Like so:

teams-for-linux --incomingCallCommand second_ringer.sh

second_ringer.sh:
#!/bin/bash
export PULSE_SINK=0
vlc -aout pulse -I dummy ring.mp3

### CODE REVIEW

**Comment by IsmaelMartinez** [comment]
Hi @93Sprinter , thanks for your contribution.

Can you increase the "package.json" version number (just the last number) and then I can create a pre-release and see if people can validate and enjoy this new feature. 

Again, thanks a lot!

**Comment by 93Sprinter** [comment]
Hi @IsmaelMartinez ,
I merged your branch into my fork and increased the version to 1.3.20. I hope that was correct.

**Comment by IsmaelMartinez** [comment]
> Hi @IsmaelMartinez , I merged your branch into my fork and increased the version to 1.3.20. I hope that was correct.

that was perfect. I am going to release 1.3.20 with the other PR that was merged, but I will make the changes and also release this as a pre-release. 

Thanks for contributing!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.3.19",
+  "version": "1.3.20",
```
Comment (IsmaelMartinez) [inline package.json]:
```suggestion
  "version": "1.3.21",
```

**Comment by IsmaelMartinez** [comment]
Pre-release is ready:

https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.3.21

I have created also a discussion page in https://github.com/IsmaelMartinez/teams-for-linux/discussions/1019

Thanks again!

**Pull request merged**